frontend/views/containers/group-settings/GroupCreationModal.vue

```diff
@@ -132,6 +132,11 @@ export default {
         groupName: '',
         groupPicture: '',
         sharedValues: null,
+
+        // randomize to reduce choice bias
+        ruleOrder: Math.round(Math.random()) === 1 ? ['threshold', 'disagreement'] : ['disagreement', 'threshold'],
```

Comment:
Can you use the constants `RULE_DISAGREEMENT` and `RULE_THRESHOLD` here instead? (There are other places this needs to be done, but I won't point them out per your request ^_^)

Also, can you rename `RULE_THRESHOLD` to `RULE_PERCENTAGE` throughout the project so that we can also close #897?